src/pages/Dashboard/sub-pages/Vaults/VaultsTable/index.tsx

```diff
@@ -214,30 +277,64 @@ const VaultsTable = (): JSX.Element => {
         }
       }
     ],
-    [t, secureCollateralThreshold]
+    [t]
   );
 
   const vaults: Array<Vault> | undefined = React.useMemo(() => {
     if (
       vaultsExt &&
-      btcToCollateralTokenRate &&
-      liquidationCollateralThreshold &&
-      secureCollateralThreshold &&
+      btcToRelayChainNativeTokenRate &&
+      btcToGovernanceTokenRate &&
+      relayChainNativeTokenCollateralLiquidationThreshold &&
+      governanceTokenCollateralLiquidationThreshold &&
+      relayChainNativeTokenCollateralSecureThreshold &&
+      governanceTokenCollateralSecureThreshold &&
```

Comment:
@nud3l @tomjeatt 
This is what I mentioned about conditions being increased in a single component.
If we add more collaterals, the relevant conditions have to be added as well here and there. If we use individual tables, the table component can be reused with different props like collateral type with the same structure and side effects. So we can reflect another collateral support just by adding one more table at an upper level with the relevant collateral type. Maintaining can be easier, although it might be different from a UX perspective.

Reply:
Hi @tomjeatt 
Yes, that sounds good. Actually, I investigated it and found a solution with `useQueries`. This PR is for an MVP with two collateral tokens for now so I'm going to update the workflow with `useQueries` via the next iteration. Does it sound good?

Reply:
@crypto-engineer — yes that sounds good. We have a solution for this in place already for the vaults using a hook (see https://github.com/interlay/interbtc-ui/blob/master/src/utils/hooks/api/use-get-vaults.tsx) which is the pattern we should follow for fetching data now. In terms of priorities, can you add a ticket for this to sprint 40? Not sure whether we want to bring this in before we introduce a new collateral token or whether this waits until the next time we need to add one. Something to talk through in the backlog session on Thursday.

Reply:
@tomjeatt 
OK, let me create an improvement ticket. Could you please continue to review this PR?